src/AddTodoForm.js

```diff
@@ -0,0 +1,18 @@
+import React from "react"; 
+let AddTodoForm=(props)=>{
+let handleAddTodo=(e)=>{
+    e.preventDefault();
+    let todoTitle=e.target.title.value;
+    console.log(todoTitle);
+    props.onAddTodo(todoTitle);
+    e.target.title.value="";
```

Comment:
Only think I'll suggest is try using `form.reset()`. I think you'll figure out how to use this

Reply:
thank you for your suggestion. 
e.target.reset();